### PR TITLE
Change UI

### PR DESCRIPTION
**Pull request by bryanwongweiheng**
for merging

### CODE REVIEW

File: src/main/java/seedu/address/model/calendarevent/DateTime.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class DateTime {
 
     public static final String MESSAGE_DATETIME_CONSTRAINTS =
-            "The valid input will be year-month-localDateTime hour:minute";
+        "The valid input will be year-month-localDateTime hour:minute";
```
Comment (SleepySanjinLi) [inline src/main/java/seedu/address/model/calendarevent/DateTime.java]: will it be year-month-day hour:minute?

Comment (bryanwongweiheng) [reply src/main/java/seedu/address/model/calendarevent/DateTime.java]: hmm looks like it hasn't been updated yet

Comment (SleepySanjinLi) [reply src/main/java/seedu/address/model/calendarevent/DateTime.java]: I think no, but don't need to change la. Cuz vig has changed.

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 2]
```diff
@@ -81,57 +71,55 @@ public Stage getPrimaryStage() {
...
+//         * For now, we add following event filter to capture such key events and open
+//         * help window purposely so to support accelerators even when focus is
+//         * in CommandBox or ResultDisplay.
+//         */
+//        getRoot().addEventFilter(KeyEvent.KEY_PRESSED, event -> {
+//            if (event.getTarget() instanceof TextInputControl && keyCombination.match(event)) {
+//                menuItem.getOnAction().handle(new ActionEvent());
+//                event.consume();
+//            }
+//        });
```
Comment (SleepySanjinLi) [inline src/main/java/seedu/address/ui/MainWindow.java]: does these has not determined now?

Comment (bryanwongweiheng) [reply src/main/java/seedu/address/ui/MainWindow.java]: what do you mean?

**Pull request merged**